src-java/floodlightrouter-topology/floodlightrouter-storm-topology/src/main/java/org/openkilda/wfm/topology/floodlightrouter/bolts/SpeakerToNetworkProxyBolt.java

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2019 Telstra Open Source
```

Comment:
2020

Reply:
Hm... local git show it as rename 
```
 .../main/java/org/openkilda/wfm/topology/floodlightrouter/bolts/{RequestBolt.java => ControllerToSpeakerProxyBolt.java}
```

changed to 2020.